src/Components/CountdownTimer/CountdownTimer.js

```diff
@@ -35,23 +35,15 @@ const CountdownTimer = () => {
             time: 0,
             callback: () => timerDone(),
           },
-          /* These colors can be found in our variables.scss file (salmon progression) */
-          {
-            time: settings.workInterval * 45000,
-            callback: () => countdownColorChange("#e19e9b"),
-          },
-          {
-            time: settings.workInterval * 30000,
-            callback: () => countdownColorChange("#f06b64"),
-          },
-          {
-            time: settings.workInterval * 15000,
-            callback: () => countdownColorChange("#e44f47"),
-          },
         ]}
       >
-        {({ start, pause, reset, getTime}) => (
+        {({ start, pause, getTime, getTimerState, setTime }) => (
           <>
+            {
```

Comment:
Y'all are geniuses for thinking of this solution!  Amazing work!

Reply:
I'll miss that particular animation, but I'm so glad we got this fix!

Reply:
So glad you got this bug figured out!